### PR TITLE
Portfolio mobile version setup

### PR DESCRIPTION
**Pull request by Mikelobi**
**Portfolio mobile version setup**

In this project:
- **I created the index page.**
- **I added quality style to the project.**
- **I created linter to check errors in the project for standard product.**
- **I created gitignore file to ignore files that are not needed in the project.**
- **I added a Readme file for further details.**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <header class="bar-nav">
+        <a href="index.html" class="logo">My Logo</a>
+        <nav><img src="image/Union.png" alt="Menu icon"></nav>
+      </header>
+      <div class="container">
```
Comment (Amrhub) [inline index.html]: - [ ] Let's make sure this is a `section` tag better HTML semantic section tag should be the parent of div tag, not the other way around.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
+        <section class="detail two">
```
Comment (Amrhub) [inline index.html]: - [ ] Let's make sure our introduction paragraph is a `p` tag for better HTML semantic

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Mikelobi,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Highly organized and clean code ✔️ 
## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mikelobi) [reply index.html]: I have updated it

File: index.html (hunk 2 above)
Comment (Mikelobi) [reply index.html]: I have made changes to it

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Mikelobi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- Please make the changes the previous reviewer asked your code is still the same 
image

this is the code that I found in the branch `Portfolio-Mobile-Version-Setup`. 
- Kindly change the `div` that takes class `container` to be `section` instead of `div` for sementic.
- Please change the `section` that takes classes ` detail ` , `two` to be `p` instead of `section` for semantic



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Mikelobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**